### DISCUSSION TITLE
Add sale success status to widget closeListener function

### DISCUSSION THREAD

**Discussion by croakedile** [pretix/pretix#3253] [Ideas]
It would be useful if one could provide different behavior based on the success of a sale when the widget gets closed.
Specifically I want to be able to run a function only when sale has successfully gone through.

**Comment by wiffbi** [comment]
I think this is a nice idea, but probably hard to implement due to cross-domain limitations. The close button triggers the close-event inside the page hosting the widget, but usually checkout happens inside an iframe, which sits on a different domain and thus cannot be „accessed“ though JS. Might be possible with `window.postMessage` inside the iframe though, then translating this to the current close-listener-callback.
